src/main/resources/assets/computercraft/lua/rom/programs/shell.lua

```diff
@@ -440,6 +441,17 @@ function shell.aliases()
     return tCopy
 end
 
+function shell.setPackagePath( sPath )
+    if type( sPath ) ~= "string" then
+        error( "bad argument #1 (expected string, got " .. type( sPath ) .. ")", 2 )
+    end
+    sPackagePath = sPath
+end
+
```

Comment:
This function would not change contents of package path of currentvrunning program. It will only affect next ones.